React/Week1/my-app-week1-task/components/ProfileImage-Component/ProfileImage.jsx

```diff
@@ -0,0 +1,6 @@
+export const ProfileImage = () => {
+	return (
+		<img
+			src="/" alt="Profile" className="profile-image"/>
```

Comment:
src attribute should point to either file location or url for the image to load.

example - src="www.google/images/profile-pic-123.com" or src = "../public/profile-pic.png"